src/converters/utils/prepareResponseTime.ts

```
@@ -17,6 +18,42 @@ export const prepareResponseTime = (
 ) => {
   if (!events) return;
   const responses = getResponses(events);
+  const user = pullRequest?.user.login || invalidUserLogin;
+  ["total", user, ...(teams[user] || [])].forEach((userKey) => {
+    [dateKey, "total"].forEach((key) => {
+      const awaitingResponse = Object.values(
+        responses as Record<string, any[][]>
+      )
+        .reduce((acc, el) => {
+          const repeatedResponses = el.filter((_, index) => index > 0);
+          return [...acc, ...repeatedResponses];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)